motr/st/utils/sns_repair_shutdown.sh

```diff
@@ -98,4 +98,4 @@ main()
 
 trap unprepare EXIT
 main
-report_and_exit sns-repair-shutdown $?
+report_and_exit userspace-sns-repair-shutdown $?
```

Comment:
Is this change required after the regex fix ?

Reply:
yes, as per my understanding from our discussion we will prefix the userspace tests explicitly. Otherwise we will not be able to differentiate between kernel vs userspace tests if they are named similarly.